src/Voron/Util/Settings/PathSettingBase.cs

```diff
@@ -92,7 +92,9 @@ public static string ToFullPath(string inputPath, string baseDataDirFullPath = n
             if (PlatformDetails.RunningOnPosix)
                 result = PosixHelper.FixLinuxPath(result);
 
-            return Path.GetFullPath(result); // it will unify directory separators and sort out parent directories
+            return result != string.Empty || resultRoot == null ? 
```

Comment:
Does it affect Windows?

Reply:
Yes, there was throwing `ArgumentException` inside `Path.GetFullPath()` on each typed extra `'/'` or `'\'`